base/system/services/rpcserver.c

```diff
@@ -5148,6 +5149,11 @@ RChangeServiceConfig2A(
             ptr = lpServiceDescriptionW;
             InfoW.psd = lpServiceDescriptionW;
         }
+        else
+        {
+            /* Merge NULL lpDescription case too */
+            InfoW.psd = NULL;
```

Comment:
Looking at this again, I think I would prefer if we just zero-initialized the structure:
`SC_RPC_CONFIG_INFOW InfoW = { 0 };`

Reply:
Done. I like that too.